Credify/Credify/Utils/ValidationUtils.swift

```diff
@@ -0,0 +1,120 @@
+        return !trimmedCountryCode.isEmpty && trimmedNumber.count >= 8 && trimmedNumber.count <= 12
+    }
+    
+    static func canShowMyPage(from: UIViewController, user: CredifyUserModel) -> Bool {
+        let tableName = "serviceX"
+        var errorMessage = ""
+        
+        // Phone number
+        if !ValidationUtils.isPhoneValid(countryCode: user.countryCode, phoneNumber: user.phoneNumber) {
+            errorMessage.append(String(format:"FieldIsInvalid".localized(tableName: tableName), "\'Phone number\'"))
+            errorMessage.append(" ")
+        }
+        
+        if !errorMessage.isBlankOrEmpty {
+            UIUtils.alert(
+                from: from,
+                title: "Error".localized(tableName: tableName),
+                errorMessage: errorMessage,
+                actionText: "OK".localized(tableName: tableName)
+            )
+            return false
+        }
+        
+        return true
+    }
+    
+    static func canShowDetailPage(from: UIViewController, user: CredifyUserModel, marketId: String) -> Bool {
+        let tableName = "serviceX"
+        var errorMessage = ""
+        
+        // Market id
+        if marketId.isBlankOrEmpty {
+            errorMessage.append(String(format:"FieldIsRequired".localized(tableName: tableName), "\'Market Id\'"))
+            errorMessage.append(" ")
+        }
+        
+        // Phone number
+        if !ValidationUtils.isPhoneValid(countryCode: user.countryCode, phoneNumber: user.phoneNumber) {
+            errorMessage.append(String(format:"FieldIsInvalid".localized(tableName: tableName), "\'Phone number\'"))
+            errorMessage.append(" ")
+        }
+        
+        if !errorMessage.isBlankOrEmpty {
+            UIUtils.alert(
+                from: from,
+                title: "Error".localized(tableName: tableName),
+                errorMessage: errorMessage,
+                actionText: "OK".localized(tableName: tableName)
+            )
+            return false
+        }
+        
+        return true
+    }
+    
+    static func checkBNPLAvailable(
+        from: UIViewController,
+        offers: [OfferData],
+        providers: [Organization]
+    ) -> Bool {
+        let tableName = "serviceX"
+        
+        let isBNPLAvailable = !offers.isEmpty || !providers.isEmpty
+        if !isBNPLAvailable {
+            print("BNPL is not available for this user. You should call BNPL().getBNPLAvailability function to check it.")
+            UIUtils.alert(
+                from: from,
+                title: "Error".localized(tableName: tableName),
+                errorMessage: "BNPLIsNotAvailable".localized(tableName: tableName),
+                actionText: "OK".localized(tableName: tableName)
+            )
+            return false
+        }
+        
+        return true
+    }
+    
+    static func canStartToRedeemOffer(from: UIViewController, user: CredifyUserModel) -> Bool {
```

Comment:
showErrorIfOfferCannotStart

Reply:
I renamed it.

@ngo275 in the last commit, I created `Language` enum, added logic to show `first/last name` order